docker/Dockerfile.rocm-deps-only

```diff
@@ -63,6 +63,22 @@ RUN wget https://apt.llvm.org/llvm.sh \
 ENV CC=/usr/bin/clang-19
 ENV CXX=/usr/bin/clang++-19
 
+
+# Installing ccache here enables us to use for compiling DGL later. The cache
+# can be persisted in either a `RUN` step or `docker run` via a bind mount
+# (https://docs.docker.com/engine/storage/bind-mounts/). The image already has
+# sccache installed, but it's an ancient version that appears to have no logging
+# capabilities and I was getting cache misses on extremely slow recompiles of
+# DGL. Even installing a newer version, I still had the cache misses and not
+# much indication of what was wrong. With ccache OTOH it took about 5 minutes to
+# debug (the pcg library uses the __TIME__ macro which always busts the cache)
+# and fix (CCACHE_SLOPPINESS=time_macros).
+WORKDIR /ccache
+RUN wget https://github.com/ccache/ccache/releases/download/v4.11.2/ccache-4.11.2-linux-x86_64.tar.xz \
```

Comment:
Why installing like this instead of something like apt install? (I'm not opposed to this)